.github/workflows/compatibility.yml

```diff
@@ -2,7 +2,7 @@ name: compatibility
 on:
   push:
     branches:
-      - 'compatibility'
+      - "compatibility"
```

Comment:
In YAML, uninterpreted strings are represented using single quotes, while double-quoted strings enable the interpretation of escaped characters.

Strings without quotes are considered to be uninterpreted strings by default. It is generally recommended to use quotes (single or double) around strings

I know, it's a detail… 😁

Reply:
there's no functional difference between using single quotes or double quotes. Both will work just as well, but YAML parsers, and by extension some formatters, consider some strings as special cases. In general, when a string contains characters like `  /, **,  :, -,  ` etc., these strings should be enclosed in quotes to avoid misinterpretation by the parser.

``` yml
name: deploy
on:
  push:
    branches:
      - "master"
      - "hotfix/**"

jobs:
  binary:
    runs-on: ${{ matrix.os }}
    strategy:
      max-parallel: 3
      matrix:
        os: [windows-latest, ubuntu-20.04]

    steps:
      - name: 🐙 Checkout GitHub repo (+ download lfs dependencies)
        uses: actions/checkout@v3
        with:
          fetch-depth: 0

      - name: 🔗 Install wget for Windows
        if: matrix.os == 'windows-latest'
        run: choco install wget --no-progress
```

In this example the strings under branches ("hotfix/**") include a /, which is a special character in YAML, and therefore the formatter automatically encloses these strings in double quotes. On the other hand, the strings that remain with single quotes, like 'windows-latest', do not contain any special characters, so they don't require double quotes.